core-java-modules/core-java-arrays-guides/src/main/java/com/baeldung/printarrays/PrintArrayJava.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.printarrays;
+import java.util.Arrays;
+
+public class PrintArrayJava {
+
+    // Print array content using a for loop
+    public String printArrayUsingForLoop(String[] empArray) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < empArray.length; i++) {
+            result.append(empArray[i]).append(" ");
+        }
```

Comment:
Please use enhanced for loops.

```suggestion
        for (String s : empArray) {
            result.append(s)
                .append(" ");
        }
```

Reply:
I've already explained the enhanced for loop in the second method, which is why I used the traditional for loop here to print the array. If I switch to the enhanced for loop, both methods would be the same.